Bottom: 9d619ed5215c6596d6cdb69e00586695e39930f5
Top:    f04f7af85cf19e5b3168f57e3887e9e69246d337
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-30 00:24:39 +0100

Refresh of draw-c-expose-some-drawing-api

---

diff --git a/src/draw.c b/src/draw.c
index 93b4f7e..675f76f 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -600,7 +600,7 @@ element_callback (const BoxType * b, void *cl)
  * prints assembly drawing.
  */
 
-static void
+void
 PrintAssembly (int side, const BoxType * drawn_area)
 {
   int side_group = GetLayerGroupNumberByNumber (max_copper_layer + side);
